diff --git a/node_modules/@tiptap/react/dist/index.js b/node_modules/@tiptap/react/dist/index.js
index 937d89b..b69b88f 100644
--- a/node_modules/@tiptap/react/dist/index.js
+++ b/node_modules/@tiptap/react/dist/index.js
@@ -264,6 +264,11 @@ class ReactNodeView extends NodeView {
         this.contentDOMElement = this.node.isLeaf
             ? null
             : document.createElement(this.node.isInline ? 'span' : 'div');
+
+        if (this.options.contentDOMElementTag) {
+            this.contentDOMElement = document.createElement(this.options.contentDOMElementTag);
+        }
+
         if (this.contentDOMElement) {
             // For some reason the whiteSpace prop is not inherited properly in Chrome and Safari
             // With this fix it seems to work fine
diff --git a/node_modules/@tiptap/react/src/index.ts b/node_modules/@tiptap/react/src/index.ts
index 19e6d22..f3cec85 100644
--- a/node_modules/@tiptap/react/src/index.ts
+++ b/node_modules/@tiptap/react/src/index.ts
@@ -1,3 +1,4 @@
+export * from '@tiptap/core'
 export * from './BubbleMenu'
 export { Editor } from './Editor'
 export * from './EditorContent'
@@ -7,4 +8,3 @@ export * from './NodeViewWrapper'
 export * from './ReactNodeViewRenderer'
 export * from './ReactRenderer'
 export * from './useEditor'
-export * from '@tiptap/core'
